File: leetcode-cn-242.py
class Solution(object):
    def isAnagram(self, s, t):
        """
        :type s: str
        :type t: str
        :rtype: bool
        """
        l1 = list(s)
        l2 = list(t)
        l1.sort()
        l2.sort()
        return cmp(l1, l2)

File: leetcode-cn-cpp/leetcode-cn-242.cpp
#include <iostream>
#include <algorithm>
#include <vector>

using namespace std;

class Solution {
public:
    bool isAnagram(string s, string t) {
        vector<char> v1(s.size());
        vector<char> v2(t.size());
        copy(s.begin(), s.end(), v1.begin());
        copy(t.begin(), t.end(), v2.begin());
        sort(v1.begin(), v1.end());
        sort(v2.begin(), v2.end());
        for (int i=0; i<v1.size() && i<v2.size(); i++){
            if (v1[i] != v2[i])
                return false;
        }

        return true;
    }
};

int main(){
    return 0;
}